code-of-conduct.md

```diff
@@ -0,0 +1,18 @@
+## Kubernetes Community Code of Conduct
+
+The Kubernetes open source community and project is a professional community dedicated to advancing the
+state of the art in container cluster management.  Participants in the Kubernetes community are
+expected to behave according to professional standards and in accordance with their employer's policies
+on appropriate workplace behavior.
+
+While participating in the Kubernetes open source project, Kubernetes events, or related social networking
+opportunities, participants should not engage in discriminatory or offensive speech or actions regarding
+gender, sexuality, race, or religion. At Kubernetes events or meet-ups, speakers should be especially
+aware of these concerns.
+
+The Kubernetes team does not condone any statements by speakers contrary to these standards.  The
+Kubernetes team reserves the right to deny participation and/or eject from an event (without refund) any
+individual found to be engaging in discriminatory or offensive speech or actions.
+
+Please bring any concerns to to the immediate attention of the Kubernetes team.
```

Comment:
Agree -- having some very specific place for stuff to go would be great.  To start it might be a couple of people (I'm happy to volunteer) but perhaps we should have an abuse@kubernetes.io alias set up at some point.